tests/kryoptic.nss-init.sh

```diff
@@ -31,3 +31,5 @@ export TESTPORT="29000"
 export SUPPORT_ALLOWED_MECHANISMS=0
 # not yet
 export SUPPORT_ML_DSA=0
+# not yet
+export SUPPORT_ML_KEM=0
```

Comment:
Why the NSS DB should not support these?

Reply:
because kryoptic was broken, we just merged https://github.com/latchset/kryoptic/pull/350 to unbreak it but until we get it into fedora it won't work.

Reply:
Don't we run the kryoptic build from main in the CI here? https://github.com/latchset/pkcs11-provider/blob/main/.github/workflows/kryoptic.yml#L64